backend/pom.xml

```diff
@@ -17,6 +17,7 @@
         <sonar.organization>jonashonecker</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
         <java.version>22</java.version>
+        <mongodb.version>4.11.0</mongodb.version>
```

Comment:
Why do you set an explicit version for your mongo driver? And why do you not use it anywhere in your project?

Reply:
I need to use this or a higher version to use the `vectorSearch`method (see [source](https://www.mongodb.com/developer/products/atlas/java-spring-boot-vector-search/?utm_campaign=devrel&utm_source=youtube&utm_medium=organic_social&utm_content=iKwuvuTxxK0&utm_term=jay.javed))